protocol/src/main/java/org/apache/drill/exec/proto/UserBitShared.java

```diff
@@ -901,6 +905,10 @@ private FragmentState(int value) {
      * <code>LTSV_SUB_SCAN = 62;</code>
      */
     public static final int LTSV_SUB_SCAN_VALUE = 62;
+    /**
+     * <code>EXCEL_SUB_SCAN = 64;</code>
+     */
+    public static final int EXCEL_SUB_SCAN_VALUE = 64;
```

Comment:
@cgivre did you generate C++ code, protobuf generating instruction, please check step 3:
```
If changes are made to the DrillClient's protobuf, you would need to regenerate the sources for the C++ client as well.
Steps for regenerating the sources are available https://github.com/apache/drill/blob/master/contrib/native/client/
```
https://github.com/apache/drill/tree/master/protocol